public/modules/custom/helfi_rekry_structured_data/helfi_rekry_structured_data.info.yml

```diff
@@ -0,0 +1,6 @@
+name: 'HELfi Rekry Structured data'
```

Comment:
We should use `helfi_rekry_content` module for this instead. No need to create a new module for feature that is always gonna be enabled.

Reply:
Done.